tcms/templates/plan/get_cases.html

```diff
@@ -113,16 +94,6 @@
 			</span>
 			
 		</div>
-		<div class="mixbar fixed">
-			<div class="tit" style="width: 50px; float: left; height: 100%; margin: 0px; text-align: center;">Tags:</div>
-			<div style="margin-left: 50px; line-height:22px;" class="taglist">
-				{% for tag in case_own_tags %}
-				<a href="#testcases">{{ tag.name }}</a>&nbsp;&nbsp;<span class="grey">-</span>&nbsp;&nbsp;
-				{% empty %}
-				<span class="grey">No tags in cases</span>
-				{% endfor %}
-			</div>
-		</div>
```

Comment:
This entire block needs to stay. It shows all the tags (as a list) from all of the test cases currently shown.